compiler/ballerina-backend-jvm/src/main/ballerina/compiler_backend_jvm/jvm_method_gen.bal

```diff
@@ -26,6 +26,7 @@ function generateMethod(bir:Function func, jvm:ClassWriter cw, bir:Package modul
     // generate method desc
     string desc = getMethodDesc(func);
     jvm:MethodVisitor mv = cw.visitMethod(ACC_PUBLIC + ACC_STATIC, funcName, desc, (), ());
+    InstructionGenerator instGen = new(mv, indexMap, currentPackageName);
```

Comment:
Why did we need this change?

Reply:
Previously InstructionGenerator was created for each instruction (inside the loop) . Hence I moved out of the loop so we can reuse same object.